certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/CertificateElementRuleFactory.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.certificateservice.infrastructure.certificatemodel;
+
+import java.util.Arrays;
+import java.util.List;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementId;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementRule;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementRuleExpression;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementRuleType;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.FieldId;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.RuleExpression;
+
+public class CertificateElementRuleFactory {
+
+  private CertificateElementRuleFactory() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static ElementRule mandatory(ElementId id, FieldId fieldId) {
```

Comment:
Nice! 
Lets utilize this factory method for FK7211 mandatory question aswell!

Should we add a factory method for ElementRuleLimit aswell?

Reply:
Done